### ISSUE TITLE
why the training not used images from training folder?

### ISSUE THREAD

**Issue by bayudamai92** [glenn-jocher/glenn-jocher#9]
Dear @Glenn-jocher,

I have used YOLOv5 for quite a while now, and thank you for that. Today suddenly I realize that the training process does not used images from the given training folder as directed. Can you help me in this matter? Below is some result that being train.py. You can se below, that the tarin folder has 6742 images, while the val folder has 1932. In the training of first epoch the images used is 1932 (which is from the val folder).

____
train: Scanning /content/drive/MyDrive/DATASETS/train/labels... 6742 images, 0 backgrounds, 0 corrupt: 100% 6742/6742 [2:04:15<00:00,  1.11s/it]
train: New cache created: /content/drive/MyDrive/DATASETS/train/labels.cache
train: Caching images (0.9GB ram): 100% 6742/6742 [00:17<00:00, 383.37it/s]
val: Scanning /content/drive/MyDrive/DATASETS/val/labels... 1932 images, 0 backgrounds, 0 corrupt: 100% 1932/1932 [18:16<00:00,  1.76it/s]
val: New cache created: /content/drive/MyDrive/DATASETS/val/labels.cache
val: Caching images (0.3GB ram): 100% 1932/1932 [00:04<00:00, 407.43it/s]

AutoAnchor: 3.77 anchors/target, 1.000 Best Possible Recall (BPR). Current anchors are a good fit to dataset ✅
Plotting labels to runs/train/exp8/labels.jpg... 
Image sizes 224 train, 224 val
Using 2 dataloader workers
Logging results to runs/train/exp8
Starting training for 50 epochs...

      Epoch    GPU_mem   box_loss   obj_loss   cls_loss  Instances       Size
       0/49     0.709G    0.05013    0.02006    0.04846         21        224: 100% 422/422 [00:54<00:00,  7.78it/s]
                 Class     Images  Instances          P          R      mAP50   mAP50-95: 100% 61/61 [00:14<00:00,  4.24it/s]
                   all       1932       1932      0.213      0.828        0.3      0.252

**Comment by bayudamai92** [comment]
@glenn-jocher

**Comment by glenn-jocher** [comment]
Everything is correct here, all training and val images are used.

**Comment by bayudamai92** [comment]
Thank you @glenn-jocher .. Now i realized that the training is being run first, then the Class, Images, Instances, P, R, mAP50, mAP50-95 is shown based on the validation folder. Does it mean that actually i don't have to used val.py for the validation? Because at the end the results are the same. Is isn't?

**Comment by glenn-jocher** [comment]
@bayudamai92 yes that's correct! Training will automatically run a final validation, there is no need to manually validate afterward.
